temporian/beam/io.py

```diff
@@ -67,6 +68,57 @@
 PEventSet = beam.PCollection[IndexValue]
 
 
+class UserEventSetFormat(Enum):
+    """Various representations of EventSets for user generation or consumption.
+
+    When combining Temporian program with your own beam stages, select the
+    most suited user event-set format compatible with your code.
+    """
+
+    eventKeyValue = "eventKeyValue"
```

Comment:
how about s/eventKeyValue/singleEvents and s/eventSetKeyValue/indexedEvents? or something along those lines

Reply:
Sounds much better.
*Changed
*Added point to brainstorming session